.circleci/config.yml

```diff
@@ -30,6 +30,10 @@ jobs:
           name: Collapse RSEM
           command: ./scripts/run_in_ci.sh bash analyses/collapse-rnaseq/run-collapse-rnaseq.sh
 
+      - run:
+          name: Molecular Subtyping - HGG
+          command: OPENPBTA_SUBSET=0 ./scripts/run_in_ci.sh bash analyses/molecular-subtyping-HGG/run-molecular-subtyping-HGG.sh
```

Comment:
The `OPENPBTA_SUBSET=0` argument here seems to be the reason why `circleci` is failing. This argument skips the `02` subset script in which the GISTIC `broad_values_by_arm.txt` file is extracted. 

Also, upon extraction, the folder `2019-01-28-consensus-cnv` that contains the GISTIC file is within another folder with the same name, hence when attempting to read the contents of the .txt file, we get an error. 

I _believe_ the reason for this is in the `02` script where the GISTIC file is extracted: 
```
download.file(url = "https://github.com/AlexsLemonade/OpenPBTA-analysis/files/4123481/2019-01-28-consensus-cnv.zip",
              destfile = file.path(scratch_dir, "2019-01-28-consensus-cnv.zip"),
              quiet = TRUE)
unzip(file.path(scratch_dir, "2019-01-28-consensus-cnv.zip"),
      exdir = file.path(scratch_dir, "2019-01-28-consensus-cnv"),
      files = file.path("2019-01-28-consensus-cnv", "broad_values_by_arm.txt"))
gistic_df <- data.table::fread(file.path(scratch_dir,
                                         "2019-01-28-consensus-cnv",
                                         "broad_values_by_arm.txt"),
                               data.table = FALSE)
```
Here, the file path of the output directory is defined to be `scratch/2019-01-28-consensus-cnv` and the file path of the file we want is defined to be `2019-01-28-consensus-cnv/broad_values_by_arm.txt`. 
I think if we get rid of the "2019-01-28-consensus-cnv" in front of the file path to the file, then this should work.